lib/ansible/modules/cloud/scaleway/scaleway_ip.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/python
+#
+# Scaleway IP management module
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: scaleway_ip
+short_description: Scaleway IP management module
+version_added: "2.8"
+author: Remy Leone (@sieben)
+description:
+    - This module manages IP on Scaleway account
+      U(https://developer.scaleway.com)
+extends_documentation_fragment: scaleway
+
+options:
+  state:
+    description:
+     - Indicate desired state of the IP.
+    default: present
+    choices:
+      - present
+      - absent
+
+  organization:
+    description:
+      - Scaleway organization identifier
+    required: true
+
+  region:
+    description:
+     - Scaleway region to use (for example par1).
+    required: true
+    choices:
+      - ams1
+      - EMEA-NL-EVS
+      - par1
+      - EMEA-FR-PAR1
+
+  id:
+    description:
+    - id of the Scaleway IP (UUID)
+
+  server:
```

Comment:
Didn't we agree that:
- this module should not handle that
- scaleway_compute should be used instead
?

Reply:
Indeed :/ I guess we can fix it in another PR.